include/Connection.h

```diff
@@ -51,9 +51,9 @@ class Connection {
 
 
     void reset();
-    static void setRetryTimeout(int timeout);
-    static const int getRetryTimeout();
-    static void setConnectTimeout(int timeout);
+    void setRetryTimeout(int timeout);
```

Comment:
:+1:  `Interval` is more correct. The old `Timeout`  is followed after`MEMCACHED_BEHAVIOR_RETRY_TIMEOUT` in libmemcached.

This is a incompatible API change, I'll consider rename it in next large MINOR version but not this time.

Looks good,